lib/lhm/cleanup/current.rb

```diff
@@ -0,0 +1,62 @@
+module Lhm
+  module Cleanup
+    class Current
+      def initialize(run, origin_table_name, connection)
+        @run = run
+        @origin_table_name = origin_table_name
+        @connection = connection
+      end
+
+      attr_reader :run, :origin_table_name, :connection
+
+      def execute
+        if run
+          drop_lhm_triggers_for_origin
+          drop_lhmn_tables_for_origin
+        else
+          report
+        end
+      end
+
+      private
+
+      def drop_lhm_triggers_for_origin
+        lhm_triggers_for_origin.each do |trigger|
+          connection.execute("drop trigger if exists #{trigger}")
+        end
+      end
+
+      def lhm_triggers_for_origin
+        @lhm_triggers_for_origin ||= all_triggers_for_origin.select { |name| name =~ /^lhmt/ }
+      end
+
+      def all_triggers_for_origin
+        @all_triggers_for_origin ||= connection.select_values("show triggers like '%#{origin_table_name}'").collect do |trigger|
+          trigger.respond_to?(:trigger) ? trigger.trigger : trigger
```

Comment:
I realize this isn't your code but wtf? I wonder if this is `Mysql` vs `Mysql2` stuff.

Reply:
That's likely the cause.

Reply:
that is exactly what it is, yes.

Reply:
@jordanwheeler do you happen to know which is the `mysql2` syntax?